README.md

```diff
@@ -30,7 +30,7 @@ To address this, we're releasing this bridge package, to allow users to do
 
 ```js
 peerDependencies: {
-  "babel-core": "6.x || ^7.0.0-0"
+  "babel-core": "6.x || ^7.0.0-bridge.0"
```

Comment:
I'm not 100% for if `peerDependencies` should contain `bridge` or if `^7.0.0-0` is good enough. I believe the `7.0.0-beta` versions are incompatible, so I went with the bridge-specific peer dependency.